src/Draggable/Emitter/Emitter.js

```diff
@@ -66,7 +66,14 @@ export default class Emitter {
     }
 
     if (caughtErrors.length) {
-      throw new Error(`Draggable caught errors while triggering '${event.type}'`, caughtErrors);
+      /* eslint-disable no-console */
+
+      console.error(
+        `Draggable caught errors while triggering '${event.type}'`,
+        caughtErrors,
+      );
+
+      /* dislint-disable no-console */
```

Comment:
`dislint`?

Reply:
Also, did `prettier` want you to add the spacing around these comments?

Reply:
Before eslint wanted me to add a new line in-between, now with prettier it removes it 😄

Reply:
Yea... some of their opinions conflict, but `prettier` takes precedence.